### PR TITLE
allow graphql@0.9.0 as peerDependency

### LINKED ISSUE #277: Need to update graphql-js peerDependency to 0.9.0
in package JSONs.
Anyone is welcome to submit a PR, especially first-time contributors! 🙂

### PR DESCRIPTION
**Pull request by Chris-R3**
TODO:

- [x] ~~Update CHANGELOG.md with your change (include reference to issue & this PR)~~
- [x] ~~Make sure all of the significant new logic is covered by tests~~
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

Closes #277

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * run batched requests in parallel ([@DxCx](https://github.com/DxCx)) on [#273](https://github.com/apollostack/graphql-server/pull/273)
 * Fix GraphiQL options variables. Issue #193. ([@alanchristensen](https://github.com/alanchristensen)) on
 [PR #255](https://github.com/apollostack/apollo-server/pull/255)
+* Allow graphql@0.9.0 as peerDependency [#277](https://github.com/apollostack/graphql-server/issues/277)
```
Comment (DxCx) [inline CHANGELOG.md]: Don't you want to add your github profile link?

**Comment by DxCx** [comment]
Thanks @Chris-R3 
can you please remove the older versions as well?
only 0.8.0 || 0.9.0 should stay.

**Comment by Chris-R3** [comment]
OK the old version numbers are gone

**Comment by DxCx** [comment]
Thanks alot!

**Pull request merged**